outlook/src/classes/Lead.ts

```diff
@@ -2,32 +2,43 @@ class Lead {
     id: number;
     name: string;
     expectedRevenue: string; // string because formatted
-    logged: boolean;
+    probability: number;
+    recurringRevenue?: string;
+    recurringPlan?: string;
```

Comment:
It is a m2o in odoo (aka int), not sure about its use here, just pointing it out..

Reply:
This field is only used to display info about a lead's revenues in case it has a recurring plan, since we are only displaying these values and despite being an m2o, I think that the string type is the most suited type for now.